### ISSUE TITLE
Missing description on the --skip-preprocessing option in ilab model train help screen

### ISSUE THREAD

**Issue by ktam3** [instructlab/instructlab#1924]
`[root@nvd-srv-03 ~]# ~/ilab model train --help
Usage: ilab model train [OPTIONS]

  Takes synthetic data generated locally with `ilab data generate` and the
  previous model and learns a new model using the MLX API. On success, writes
  newly learned model to {model_dir}/mlx_model, which is where `chatmlx` will
  look for a model.

Options:
  --data-path PATH                Base directory where data is stored.
                                  [required; default:
                                  /root/.local/share/instructlab/datasets;
                                  config: 'train.data_path']
  --ckpt-output-dir PATH          output directory to store checkpoints in
                                  during training  [required; default:
                                  /root/.local/share/instructlab/checkpoints;
                                  config: 'train.ckpt_output_dir']
  --data-output-dir PATH          output directory to store training data in
                                  [required; default:
                                  /root/.local/share/instructlab/internal;
                                  config: 'train.data_output_dir']
  --input-dir PATH                Path to generated files to use as input.
  --gguf-model-path TEXT          Local directory where gguf model is stored.
  --skip-preprocessing
  --tokenizer-dir PATH            Base directory where tokenizer is stored.
  --model-path PATH               HuggingFace model repo path, in the format
                                  of <namespace>/<repo_name>.  [required;
                                  default: instructlab/granite-7b-lab; config:
                                  'train.model_path']
  --iters INTEGER                 Number of iterations to train LoRA.
  --local                         Whether or not `model_dir` is remote from
                                  HuggingFace.
  -sq, --skip-quantize            Whether to skip quantization while
                                  converting to MLX. This parameter will be
                                  ignored if --gguf-model-path and
                                  --tokenizer-dir are specified.
  --num-epochs INTEGER            The number of times the training data is
                                  passed through the training algorithm.
                                  Please note that this value is used on Linux
                                  platforms only.  [required; default: 10;
                                  config: 'train.num_epochs']
  --device [cpu|cuda|hpu]         PyTorch device for Linux training. Use
                                  'cuda' for NVidia CUDA / AMD ROCm GPU, to
                                  use specific GPU, set visible GPU before run
                                  train command.  [default: cpu]
  --max-seq-len INTEGER           maximum length, in tokens, of a single
                                  sample.  [required; default: 4096; config:
                                  'train.max_seq_len']
  --max-batch-len INTEGER         maximum overall length of samples processed
                                  in a given batch.  [required; default:
                                  10000; config: 'train.max_batch_len']
  --effective-batch-size INTEGER  total batch size across all GPUs  [required;
                                  default: 3840; config:
                                  'train.effective_batch_size']
  --save-samples INTEGER          The number of samples processed in between
                                  checkpoints.  [required; default: 250000;
                                  config: 'train.save_samples']
  --learning-rate FLOAT           learning rate for training  [required;
                                  default: 2e-05; config:
                                  'train.additional_args.learning_rate']
  --warmup-steps INTEGER          warmup steps for training  [required;
                                  default: 25; config:
                                  'train.additional_args.warmup_steps']
  --deepspeed-cpu-offload-optimizer BOOLEAN
                                  if true enables optimizer offload
                                  [required; default: False; config:
                                  'train.deepspeed_cpu_offload_optimizer']
  --deepspeed-cpu-offload-optimizer-ratio FLOAT
                                  cpu offload optimizer ratio  [required;
                                  default: 1; config: 'train.additional_args.d
                                  eepspeed_cpu_offload_optimizer_ratio']
  --deepspeed-cpu-offload-optimizer-pin-memory BOOLEAN
                                  if true pin memory when using cpu optimizer
                                  [required; default: False; config: 'train.ad
                                  ditional_args.deepspeed_cpu_offload_optimize
                                  r_pin_memory']
  --lora-rank INTEGER             rank of update matricies  [required;
                                  default: 4; config: 'train.lora_rank']
  --lora-alpha INTEGER            how influential/strong lora tune will be
                                  [required; default: 32; config:
                                  'train.additional_args.lora_alpha']
  --lora-dropout FLOAT            dropout for LoRA layers  [required; default:
                                  0.1; config:
                                  'train.additional_args.lora_dropout']
  --lora-target-modules TEXT      LoRA modules to use  [default: q_proj,
                                  k_proj, v_proj, o_proj; config:
                                  'train.additional_args.lora_target_modules']
  --lora-quantize-dtype TEXT      quantization data type to use when training
                                  a LoRA.
  --is-padding-free BOOLEAN       whether or not we are training a padding
                                  free transformer.  [default: False; config:
                                  'train.is_padding_free']
  --gpus INTEGER                  this is the number of GPUs to use. This is a
                                  torch specific arg and must be called nproc-
                                  per-node  [default: 1; config:
                                  'train.nproc_per_node']
  --nnodes INTEGER                number of machines in the training pool.
                                  [required; default: 1; config:
                                  'train.additional_args.nnodes']
  --node-rank INTEGER             the rank of this machine in the training
                                  group.  [required; default: 0; config:
                                  'train.additional_args.node_rank']
  --rdzv-id INTEGER               this is the training group ID. So, if there
                                  are multiple matching endpoints, only the
                                  machines with matching IDs can connect.
                                  [required; default: 123; config:
                                  'train.additional_args.rdzv_id']
  --rdzv-endpoint TEXT            this is the rendezvous endpoint which other
                                  torchrun jobs will join on.  [required;
                                  default: 127.0.0.1:12222; config:
                                  'train.additional_args.rdzv_endpoint']
  --legacy                        if true, enables the legacy linux training
                                  codepath from release 0.17.0 and prior.
  --help                          Show this message and exit.
[root@nvd-srv-03 ~]# ~/ilab --version
ilab, version 0.18.0b1`
